ArduCopter/heli.cpp

```diff
@@ -183,6 +182,20 @@ void Copter::heli_update_rotor_speed_targets()
                 motors.set_desired_rotor_speed(0);
             }
             break;
+        case ROTOR_CONTROL_MODE_CLOSED_LOOP_POWER_OUTPUT:
+            // pass setpoint through as desired rotor speed
+            if (rsc_control_deglitched > 0) {
+                motors.set_interlock(true);
+                if ((rpm_sensor.get_rpm(0) > 0) && ((heli_flags.using_governor_switch && heli_flags.governor_enable) || !heli_flags.using_governor_switch)){
+                    motors.set_rsc_governor_enabled(true, motors.get_gov_rpm_setpoint(), rpm_sensor.get_rpm(0));
```

Comment:
I would have 'enabled' as the only parameter for the 'set_rsc_governor_enabled' method. The RPM set-point is already present in the AP_MotorsHeli implementation, and I think it would be cleaner to add 'rpm_sensor' as a constructor argument to AP_MotorsHeli. That way, each AP_MotorsHeli implementation (hint hint :smile:) could use the RPM sensor as needed.